### ISSUE TITLE
K8s deployment problem

### ISSUE THREAD

**Issue by strogonoff** [ietf-ribose/bibxml-service#195]
The error occurs at the “wait for migration” stage:

```
Run kubectl wait --for=condition=complete --timeout=300s job/db-migration
error: exec plugin: invalid apiVersion "client.authentication.k8s.io/v1alpha1"
Error: Process completed with exit code 1.
```

**Comment by ronaldtse** [comment]
@kwkwan can you please help here? Thanks.

**Comment by strogonoff** [comment]
(Deployment seems to be working again as of today.)

**Comment by ronaldtse** [comment]
@strogonoff then can we close this?

**Comment by strogonoff** [comment]
I’m not qualified to say, @kwkwan could close if he’s finished making relevant changes…

**Comment by kwkwan** [comment]
Closed by dd618c0
